src/app/Command.h

```diff
@@ -75,13 +75,7 @@ class Command
      *  @retval #CHIP_NO_ERROR On success.
      *
      */
-    CHIP_ERROR Init(Messaging::ExchangeManager * apExchangeMgr, InteractionModelDelegate * apDelegate);
-
-    /**
-     *  Shutdown the Command. This terminates this instance
-     *  of the object and releases all held resources.
-     */
-    void Shutdown();
+    CHIP_ERROR Init(Messaging::ExchangeManager * apExchangeMgr);
```

Comment:
This removes the `InteractionModelDelegate` on the server side too - does this cause any unintended consequences?

Reply:
After going through the code, CommandHandler does not use IMDelegate.

Regarding #9460, future callbacks will using seperate callback class.

Reply:
After going through the code, CommandHandler does not use IMDelegate.

Regarding #9460, future callbacks will using seperate callback class.